scalanative-stubs/src/main/scala/scala/scalajs/js/annotation/JSExportDescendentClasses.scala

```diff
@@ -0,0 +1,6 @@
+package scala.scalajs.js.annotation
+
+class JSExportDescendentClasses(ignoreInvalidDescendants: Boolean)
```

Comment:
This is one is deprecated in Scala.js and we should keep it being deprecated in native.